README.md

```diff
@@ -30,28 +30,111 @@ pip install -r requirements-cuda.txt
 
-1. Create a HuggingFace account from <https://huggingface.co/>
-2. Create a User Access Token by following steps in <https://huggingface.co/docs/hub/security-tokens>
-3. Download the `huggingface_hub` library and login through `huggingface-cli` with your User Access Token by following the steps in <https://huggingface.co/docs/huggingface_hub/quick-start>
 
-#### Benchmarking Datasets Setup
+### Benchmarking Datasets Setup
 
-HuggingFace datasets will download into `${HF_HOME}/datasets/` once HuggingFace Hub access is setup.
 
-COCO: is downloaded automatically from <https://cocodataset.org>, no login is needed. The cache location is `~/.cache/coco`
+To set up the three required datasets for running benchmarking tests within this repository, follow these steps for each dataset:
 
-LGG Segmentation Dataset: must be manually downloaded from <https://www.kaggle.com/datasets/mateuszbuda/lgg-mri-segmentation>, this requires a Kaggle login. The commands below are to extract the downloaded archive into the correct location: `~/.cache/mldata/lgg_segmentation/kaggle_3m`.
+1. **HuggingFace datasets**: will download into `${HF_HOME}/datasets/` once [HuggingFace Hub access](#setup-access-to-huggingface-hub) is set up.
+
+2. **COCO Dataset**: You can automatically download the COCO dataset from [here](https://cocodataset.org/#download:~:text=2017%20Val%20images%20%5B5K/1GB%5D). No login is required, and the dataset will be cached in `~/.cache/coco`.
+
+To download the COCO dataset, follow these steps:
 
 ```bash
 # use another location for MLDATA_DIR if desired, below is default
+# Create the `coco` directory inside the cache directory:
+mkdir -p ~/.cache/mldata/coco
+
+# Navigate to the `coco` directory:
+cd ~/.cache/coco
+
+# Create the `images` directory:
+mkdir images
+cd images
+
+# Download the COCO validation images:
+wget https://cocodataset.org/#download:~:text=2017%20Val%20images%20%5B5K/1GB%5D
+
+# Unzip the downloaded file:
+unzip val2017.zip
+
+# Move back to the `coco` directory:
+cd ..
+
+# Create the `annotations` directory:
+mkdir annotations
+cd annotations
+
+# Download the COCO train/val annotations:
+wget https://cocodataset.org/#download:~:text=2017%20Train/Val%20annotations%20%5B241MB%5D
+
+# Unzip the downloaded file:
+unzip annotations_trainval2017.zip
+```
+
+3. *LGG Segmentation Dataset*: must be manually downloaded from <https://www.kaggle.com/datasets/mateuszbuda/lgg-mri-segmentation>, this requires a Kaggle login. The commands below are to extract the downloaded archive into the correct location: `~/.cache/mldata/lgg_segmentation/kaggle_3m`.
+
+```bash
+# use another location for MLDATA_DIR if desired, below is default;
+# Download and move the downloaded archive and unzip within the llgg_segmentation folder.
 mkdir -p ~/.cache/mldata/lgg_segmentation
 cd ~/.cache/mldata/lgg_segmentation
-# download and move the archive here then unzip
-unzip archive.zip
```

Comment:
Sorry -- I just realized that it downloads it as `archive.zip` :(

You can add these two lines back in!

Reply:
ok I did not delete it :)